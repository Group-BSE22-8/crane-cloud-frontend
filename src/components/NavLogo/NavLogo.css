.Logo {
  width: 300px;
  padding: 0px 0px;
}
.Brand img {
  vertical-align: middle;
  border-style: none;
  box-sizing: none;
  width: 200px;
  display: inline;
}
.Brand {
  position: absolute;
  /* float: left; */
  line-height: 65px;
  text-transform: uppercase;
  font-size: 1.4em;
  display: inline-block;
  font-size: 22px;
  /* width: 200px; */
  background-color: var(--primary-color);
}
