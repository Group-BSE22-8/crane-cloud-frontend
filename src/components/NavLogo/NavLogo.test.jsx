import React from 'react';

import { shallow } from 'enzyme';

import NavLogo from '.';


describe('Test the nav logo', () => { 
    it('checks for correct rendering of the component', () => {
        const NavLogoComponent = shallow(<NavLogo />);
        expect(NavLogoComponent).toMatchSnapshot();
        expect(NavLogoComponent.find('img').length).toBe(1);
    })
})