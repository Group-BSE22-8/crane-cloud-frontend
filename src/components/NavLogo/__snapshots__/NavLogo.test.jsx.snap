// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the nav logo checks for correct rendering of the component 1`] = `ShallowWrapper {}`;
